projectable.el

```diff
@@ -453,7 +453,7 @@ Optionally called F as the function used to switch the buffer."
 (defun projectable-find-file ()
   "Call `projectable--find-file` for FILE with `find-file` as function call."
   (interactive)
-  (projectable--find-file projectable-file-alist 'find-file))
+  (if projectable-file-alist (projectable--find-file projectable-file-alist 'find-file)) (message "You do not currently have a project set. You can set a project by calling projectable-change (c-x p c)"))
```

Comment:
* You should call `(projectable-message MSG t)` which prefixes the message to let you know it's from projectable.
* You should put the if statement on two lines to make it easier to read.